ui/src/org.ts

```diff
@@ -30,16 +32,32 @@ export const nameConstraints = {
   }
 };
 
-// DUMMY DATA
+// Make sure we make a new one every time
+export const validationStore = () => createValidationStore(nameConstraints, {
+  promise: getNameAvailability,
+  validationMessage: "Sorry, this name is already taken"
+})
 
-export const org: Org = {
-  name: "",
-  avatar: {
-    imageUrl: imageUrl
+// DUMMY DATA
+// TODO(sos): Replace with actual avatar fallback request once the endpoint is ready
+export const generateAvatar = (id: string): EmojiAvatar => {
+  return {
+    background: {
+      r: 0,
+      g: 200,
+      b: 222
+    },
+    emoji: "😘"
   }
 }
 
-export const getOrg = (id: string): Promise<Org> => api.get<Org>(`orgs/${id}`)
+export const getSubject = (id: string) => {
+  return {
+    name: id,
+    kind: "org",
+    avatarFallback: generateAvatar("")
+  }
+};
 
 interface RegisterOrg {
```

Comment:
We are suffixing these with `Input` to make clear that they are used as expected shapes for API calls.